### ISSUE TITLE
ICE dilution fridge: comission/test

### ISSUE THREAD

**Issue by Tom-Willemsen** [ISISComputingGroup/IBEX#4926]
As a muon scientist, I want to be confident that the ICE fridge can be controlled via IBEX.

To do this, it would be useful to run a test against the real system in a cryogenic state.

Acceptance:

- Required functionality is documented
- Required functionality works with hardware

Notes:
- This will need to be coordinated with both cryogenics groups and the instrument scientists, to make sure that the system can be set up and cryogenically supported.
- January will probably be the best time to run these tests (out of cycle but still leaving some time if changes need to be made)
- The ICE fridge IOC is implemented in https://github.com/ISISComputingGroup/IBEX/issues/3876
- Cryogenics prefer early January as "we then have time to thermally cycle ICE before cycle starts (if needed)"

**Comment by Alexandru-Dascalu** [comment]
During the commissioning, we need to look into some things that we are not sure about the old LabVIEW software of this device. These are listed in https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/To-do-list-for-ICE-dilution-fridge-IOC-for-testing-on-actual-device . If we find out there are changes needed to the IOC we will make a separate ticket for making those changes.

**Comment by Tom-Willemsen** [comment]
The ICE fridge is apparently broken. Cryogenics are going to try and repair and test the fridge before cycle. If it can be repaired, we may be able to do both tests in parallel. But we may not be able to do this if the fridge can't be repaired.

**Comment by Tom-Willemsen** [comment]
Cryogenics section have offered us a window of opportunity for a few hours tomorrow morning to try to connect to it and run a few tests.

**Comment by Tom-Willemsen** [comment]
Tested in conjunction with Cryogenics and instrument scientist.

Issues found:
- On the OPI under the mimic tab, P3 is actually pointing at the PV for P2.
- `MC:USER` is 1000x too large (when reading resistance)
- On discussion with the scientist, we should split `MC:USER` into two distinct PVs - one that measures the temperature, one for the resistance.
- The lakeshore replies `OK` rather than `Set Lakeshore 370` in several instances
- Lakeshore mixing chamber I and D parameters can only be integers, not floats
- Units of pressure on the mimic should be `mbar` not `Pa`
- Units of proportional valve readbacks should be `%`
- In CS-Studio enable border on alarm for mode (as the mode cannot be read back from hardware)

**Comment by Alexandru-Dascalu** [comment]
I modified the device documentation in https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/ICE-Dilution-Fridge to mention the split of the MC resistance and temp in 2 readbacks on the OPI, making the Lakeshore MC I and D values to be integers and how the mode drop down works. I took out MC-R? from the list of unimplemented commands since it was implemented.

I also added a modified copy of the remote interface that correctly says that Lakeshore set replies are OK. I left the original because it is a read only file.
